src/forms/stock-dispense/stock-dispense.component.tsx

```diff
@@ -110,7 +111,6 @@ const StockDispense: React.FC<StockDispenseProps> = ({
         items={validInventoryItems}
         onChange={({ selectedItem }) => {
           updateInventoryItem(selectedItem);
-          updateAvailableQuantityToMedicationDispenseQuantity(selectedItem);
```

Comment:
To be clear, I'm fine if you want to keep this, and update the amount available to dispense... the problem I wanted to revert is that it was enforcing the amount you could dispense even if you weren't using stock management, and didn't have the restrict quantity enabled.

If you do want to remove this, shouldn't we remove the changes to dispense-form.component.tsx from your initial commit (https://github.com/openmrs/openmrs-esm-dispensing-app/commit/f1b80940fa02927e5072c535aa4217baf7b70cf9) because there is no need to create a setAvailableQuantity function in a "updateAvailalbeQuantityToMedicationDispenseQuantity" function if it no longer being used.  Or am I missing something?